modulemd/v2/include/modulemd-2.0/modulemd-module-stream.h

```diff
@@ -0,0 +1,189 @@
+/*
+ * This file is part of libmodulemd
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Fedora-License-Identifier: MIT
+ * SPDX-2.0-License-Identifier: MIT
+ * SPDX-3.0-License-Identifier: MIT
+ *
+ * This program is free software.
+ * For more information on the license, see COPYING.
+ * For more information on free software, see <https://www.gnu.org/philosophy/free-sw.en.html>.
+ */
+
+#pragma once
+
+#include <glib.h>
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+/**
+ * SECTION: modulemd-module-stream
+ * @title: Modulemd.ModuleStream
+ * @stability: stable
+ * @short_description: A parent class for all versions of #ModulemdModuleStream
+ * objects.
+ */
+
+/**
+ * ModulemdModuleStreamVersionEnum:
+ * @MD_MODULESTREAM_VERSION_ONE: Represents v1 of the #Modulemd.ModuleStream
+ * metadata format.
+ * @MD_MODULESTREAM_VERSION_TWO: Represents v2 of the #Modulemd.ModuleStream
+ * metadata format.
+ * @MD_MODULESTREAM_VERSION_LATEST: Represents the highest-supported version of
+ * the #Modulemd.ModuleStream metadata format.
+ *
+ * Since: 2.0
+ */
+typedef enum
+{
+  MD_MODULESTREAM_VERSION_UNSET = 0,
+
+  MD_MODULESTREAM_VERSION_ONE = 1,
+  MD_MODULESTREAM_VERSION_TWO = 2,
+
+  MD_MODULESTREAM_VERSION_LATEST = MD_MODULESTREAM_VERSION_TWO
+} ModulemdModuleStreamVersionEnum;
+
+
+#define MODULEMD_TYPE_DEFAULTS (modulemd_defaults_get_type ())
```

Comment:
Is this a copy-paste error from defaults?

Reply:
Looks like it. I'll drop it.